### ISSUE TITLE
vestedTransfer not working as expected for past blocks

### ISSUE THREAD

**Issue by chrlschwb** [Joystream/joystream#4581]
Tried to do a vestedTransfer between my accounts with the following parameters.

- locked : 30 JOY
- per block : 0.00004 JOY
- startingBlock : 1

![image](https://user-images.githubusercontent.com/101491745/215318007-7ac554d8-513f-4400-9870-cdfc03b4d5d7.png)

Expectation was to create a vesting that happened between blocks 1 to 750,000.
At the moment of this extrinsic transaction, the block number was 726,787.
So if my expectations were met, claimable tokens should be 29 JOY.

![image](https://user-images.githubusercontent.com/101491745/215318162-4eb857b0-c7cd-415c-83ee-062b313157f8.png)

Pioneer account info shows exactly this, but if I click "Claim", it shows I have only <0.01 JOY to claim.

I believe vestedTransfer doesn't work for past blocks. And if such is the case, the input should be restricted to stop users from making mistakes and losing funds.

**Comment by bedeho** [comment]
do you believe this is a Prioneer issue? if so it should be moved.

**Comment by mochet** [comment]
I don't think this is an error. Most of the extrinsics that allow for entering a startblock will produce strange results if you enter a block that hasn't yet happened.
